bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -29,7 +29,7 @@ eyeball-im = { workspace = true }
 extension-trait = "1.0.1"
 futures-core = { workspace = true }
 futures-util = { workspace = true }
-matrix-sdk-ui = { path = "../../crates/matrix-sdk-ui", default-features = false, features = ["e2e-encryption", "experimental-room-list", "experimental-encryption-sync", "experimental-notification-client", "experimental-sync-service"] }
+matrix-sdk-ui = { path = "../../crates/matrix-sdk-ui", default-features = false, features = ["e2e-encryption", "experimental-polls", "experimental-room-list", "experimental-encryption-sync", "experimental-notification-client", "experimental-sync-service"] }
```

Comment:
New: We consider that everything inside `matrix-sdk-ui` is experimental.
Consequence: Please remove the `experimental-polls` flag :-).

Reply:
Neat! Done.